ee/clickhouse/queries/funnels/funnel.py

```diff
@@ -97,8 +97,15 @@ def _format_single_funnel(self, result, with_breakdown=False):
                 serialized_result.update({"average_conversion_time": None, "median_conversion_time": None})
 
             if with_breakdown:
+                # breakdown will return a display ready value
+                # breakdown_value will return the underlying id if different from display ready value (ex: cohort id)
                 serialized_result.update(
-                    {"breakdown": result[-1] if isinstance(result[-1], str) else Cohort.objects.get(pk=result[-1]).name}
+                    {
+                        "breakdown": Cohort.objects.get(pk=result[-1]).name
```

Comment:
We don't seem to have any test cases where breakdown_value != breakdown below. Missing cohort tests?